lang/ko.json

```diff
@@ -0,0 +1,51 @@
+{
+  "CN.AbilityDialogWarningItem": "이 물건이나 특성을 사용하면 {item}의 집중은 끝나게 됩니다.",
+  "CN.AbilityDialogWarningSpell": "이 주문을 시전하면, {item}의 집중은 끝나게 됩니다.",
+  "CN.AbilityDialogWarningSpellLevel": "이 주문을 {level} 외의 레벨로 시전하면, {item}의 집중은 끝나게 됩니다.",
+  "CN.AbilityDialogWarningUnfocusedItem": "이 물건이나 특성을 사용할 수 있지만, 그것에 집중할수는 없습니다.",
+  "CN.AbilityDialogWarningUnfocusedSpell": "이 주문을 시전할 수 있지만, 그것에 집중할수는 없습니다.",
+  "CN.ButtonEndConcentration": "집중 끝내기",
+  "CN.ButtonRemoveTemplates": "템플릿 제거",
+  "CN.ButtonSavingThrow": "내성굴림DC {dc} {saveType}",
+  "CN.ConcentrationAbbr": "Conc",

```

Comment:
I'm not familiar with the language, so I can't speak to how abbreviations work. Is it correct to not translate abbreviations?